pypeit_files/keck_deimos_600zd_m_6500.pypeit

```diff
@@ -28,8 +28,8 @@ setup end
 data read
  path /home/xavier/local/Python/PypeIt-development-suite/RAW_DATA/keck_deimos/600ZD_M_6500/
 |           filename |                date  | frametype         |         target | exptime | binning | dichroic | dispname |     dispangle | decker |
-| d1010_0005.fits.gz | 2015-10-10T01:31:58  |     bias          |     DOME PHLAT |       1 |     1,1 |     None |    600ZD | 1097.54162598 | mask04 |
-| d1010_0006.fits.gz | 2015-10-10T01:33:11  |     bias          |     DOME PHLAT |       1 |     1,1 |     None |    600ZD | 1097.54162598 | mask04 |
+| d1010_0005.fits.gz | 2015-10-10T01:31:58  |     bias,dark     |     DOME PHLAT |       1 |     1,1 |     None |    600ZD | 1097.54162598 | mask04 |
```

Comment:
I somewhat think DEIMOS should not frametype any images as `dark`.
Looking at the code, it would seem I am right.  Am I missing something?

Reply:
Okay, you caught me being lazy.  I agree, but have a look here: https://github.com/pypeit/PypeIt/blob/db3f2309838adc3797399e69389bb3b76b251a85/pypeit/metadata.py#L1242

Currently, anything that is designated as a `bias` but also has an exposure time that is greater then a minimum exposure time defined for each spectrograph (I don't know why we have such a thing) is designated as a `dark` frame.  So these frames get flagged as darks.  To my mind, those lines of code are in the wrong place, and I imagine they were a hack to get something to work.  So I took the cheap way out.  I could put in another hack that changes `minexp` for DEIMOS, but I think we would prefer to deal with the root of the issue.

Reply:
Ah, I see.  Let me open a dialogue on Slack.

Reply:
I've removed the relevant lines in pypeit and remove the dark designations in this file.